config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,110 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I've turned on things that are relatively safe already, others left for future PRs